package.json

```diff
@@ -334,6 +337,8 @@
   },
   "optionalDependencies": {
     "@img/sharp-darwin-arm64": "^0.33.4",
-    "@img/sharp-linux-x64": "^0.33.4"
+    "@img/sharp-linux-x64": "^0.33.4",
+    "@nx/nx-darwin-arm64": "18.2.4",
```

Comment:
Probably not needed anymore with the changes on master. This might cause your issues.